src/components/commons/bottomSheet/actionsBottomSheet/ActionBottomSheet.styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+import { IcomCopy } from "@assets/svgs";
+
+export const ActionBottomSheetWrapper = styled.section`
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
p3) 
<img width="1624" alt="image" src="https://github.com/TEAM-BEAT/BEAT-Client/assets/58854041/12024ac8-d123-407c-a115-ce4c11d8e1ac">

신지야 모바일 뷰에서는 지금 하단에 잘 나오는데, 이게 pc에서보면 <img width="1624" alt="image" src="https://github.com/TEAM-BEAT/BEAT-Client/assets/58854041/472361f3-af07-4425-9e66-bb0bef222148">
이렇게 화면의 왼쪽 하단에 붙어서 나와 ㅠㅠ 
이 부분은 아래처럼 코드 수정하면 중앙 하단에 나올 거야! 물론 지금은 필요 없을 수도 있겠지만, 나중에 pc에서도 BottomSheet 사용한다고 생각하면 아래처럼 수정하는 게 좋을 것 같아! 
```suggestion
  left: auto;
```
ViewBottomSheet도 똑같은 현상이 발생해서 이 부분만 수정하고 머지해도 될 것 같아~!
고생했어!!!